.stickler.yml

```diff
@@ -0,0 +1,15 @@
+# add the linters you want stickler to use for this project
+linters:
+    eslint:
+      # indicate where is the config file for stylelint
+      config: './.eslintrc.json'
+  
+  # add the files here you want to be ignored by stylelint
+  files:
```

Comment:
This file is not parsing properly due to an indentation issue you have an indentation on this line which is making the yml file not to parse, just alingn this line with line one, that is no tabs or spaces in the begginig of this line and the issue shall be taken care off and we will be alble to see what stickler has to say.

Reply:
Hi @jofer86  thanks for reviewing my code,

You mean remove indention on line 5 right?